src/components/input-components.js

```diff
@@ -0,0 +1,33 @@
+// standard class with all states found in the other inputs of this work
+
+export default class Input {
```

Comment:
Why this input don't have validation?

Reply:
The validations of this class are in the child classes in both text and email, the idea was to store in them the common properties the validations as required, enable and disable, being necessary to do only the polymorphism in the child classes and adjust the following specifications